YaST/Repos/_openSUSE_Leap_42.2_Additional.xml.in

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0"?>
+<!-- Additional Repositories for openSUSE Leap 42.2
+     These are maintained by Stephan Kulow <coolo@suse.de> and Dominique Leuenberger <dimstar@suse.de> -->
+<metapackage xmlns:os="http://opensuse.org/Standards/One_Click_Install" xmlns="http://opensuse.org/Standards/One_Click_Install">
+  <group distversion="openSUSE Build Service">
+    <repositories>
+      <repository recommended="false" format="rpm-md">
+        <_name>openSUSE BuildService - LXDE</_name>
+        <_summary>Latest LXDE release</_summary>
+        <_description>The LXDE repository in the openSUSE Build Service, which provides you with the unsupported but latest version of the LX desktop environment and software.</_description>
+        <url>http://download.opensuse.org/repositories/X11:/lxde/openSUSE_Leap_42.2/</url>
+      </repository>
+      <repository recommended="false" format="rpm-md">
+        <_name>openSUSE BuildService - LibreOffice</_name>
+        <_summary>Latest stable LibreOffice release</_summary>
+        <_description>Provides you with the latest stable version of LibreOffice, the Office suite that openSUSE uses.</_description>
+        <!-- Check the version here each time for what is supposed to be set -->
+        <url>http://download.opensuse.org/repositories/LibreOffice:/5.0/openSUSE_42.2/</url>
+      </repository>
+LibreOffice repo not yet enabled for Leap 42.2 / dimstar / 20151006 -->
```

Comment:
On Wed, 2016-07-20 at 22:40 -0700, Oliver Kurz wrote:
> 
> I don't know at all what this repos lists here are about at all. What
> do I need to do?
> 
> 
This is the list that shows when you go to YaST, Software Repositories,
Add, Community Repositories, Next
Let's focus on the Main Repos for now for Leap 42.2, keeping the others
disabled is 'good enough' for now